app/Resources/views/admin2/user/form.html.twig

```diff
@@ -0,0 +1,141 @@
+{% form_theme form 'bootstrap_3_horizontal_layout.html.twig' %}
```

Comment:
You don't have to change but next time you can use shorter
`{% form_theme form with ['bootstrap_3_horizontal_layout.html.twig', 'form/fields.html.twig'] %}`